int16 target_position
int16 target_velocity
int16 target_torque
int32 encoder_total_count
